job/testjob/dataset_api.go

```diff
@@ -82,8 +82,10 @@ type DatasetAPIClientMock struct {
 			CollectionID string
 			// InstanceID is the instanceID argument value.
 			InstanceID string
-			// InstanceUpdate is the instanceUpdate argument value.
-			InstanceUpdate dataset.UpdateInstance
+			// I is the i argument value.
```

Comment:
this is generated automatially with by moq, but you are right that the `I` naming is not clear at all. I will rename the variable in the interface, and re-generate the mock, so that this name is clearer